admin_guide/sdn_troubleshooting.adoc

```diff
@@ -360,9 +360,8 @@ on this bridge.
 cluster subnet gateway address, and is used for external network
 access. OpenShift SDN configures `netfilter` and routing rules to enable access
 from the cluster subnet to the external network via NAT.
-* `vxlan0`: The OVS VXLAN device (port 1 on `br0`), which provides access to
-containers on remote nodes.
-* `vethX` (in the main netns): A Linux virtual ethernet peer of `eth0` in the docker netns.  It will be attached to the OVS bridge on one of the other ports.
+* `vxlan_sys_4789`: The OVS VXLAN device (port 1 on `br0`), which provides access to containers on remote nodes. Referred to as `vxlan0` in the OVS rules.
+* `vethX` (in the main netns): A Linux virtual ethernet peer of `eth0` in the docker netns. It will be attached to the OVS bridge on one of the other ports.
```

Comment:
is "docker netns" properly formatted? *docker* netns ??
Should "docker" be "Docker"?